main/midnight/src/ui/uicommandwindow.cpp

```diff
@@ -340,10 +340,10 @@ void uicommandwindow::setupApproachText()
     setCommandTextName(
         ID_APPROACH,
         location_flags.Is(lif_recruitchar),
-        recruitable_characters[0]);
+        recruitable_characters.size() > 0 ?  recruitable_characters[0] : 0);
```

Comment:
Initialization was failing here. rectruitable_characters array was empty, access to zero entry failed.

Reply:
Nice catch!